package.json

```diff
@@ -118,7 +118,7 @@
     }
   },
   "volta": {
-    "node": "16.15.0",
+    "node": "18.13.0",
```

Comment:
In Gatbsy v5, [the minimal node version is v18](https://www.gatsbyjs.com/docs/reference/release-notes/migrating-from-v4-to-v5/#minimal-nodejs-version-1800).